### DISCUSSION TITLE
what are bpmn component that supported and not supported by imixs?

### DISCUSSION THREAD

**Discussion by rizalalfarizi123** [imixs/imixs-workflow#877] [Q&A] [answered]
Hi @rsoika thank you for replying my previous question, i'm still comparing workflow engine that i want to use for my project there are activiti and imixs, activiti a little bit hard for newer version because they haven't updated the documentation and they don't have modeler (as far i know they archived the project). i wan't to ask what are bpmn component that supported and not supported by imixs? thank you

**Comment by rsoika** [comment, marked as answer]
Hi @rizalalfarizi123 ,
in Imixs-Workflow the following BPMN elements are relevant for the Processing-Life-Cycle:

- Task
- Start Event
- End Event
- CatchEvent
- Exclusive Gateway (see [conditional events](https://www.imixs.org/doc/modelling/howto.html#Conditional_Events))
- Parallel Gateway (see [split events / parallel workflows](https://www.imixs.org/doc/modelling/howto.html#Split_Events)
- Event Signal Definitions (for signal events)
- Event Timer Definitions (for Boundary Events)
- Data Element (to describe more complex configurations)
- Link Event Definitions
- Conditional Sequence Flows
- Association
- Pools
- Message Object

The following elements are supported by Open-BPMN but not relevant for the sequence flow in a processing cycle

- Event Gateway
- Complex Gateway
- All specified Task Elements
- Lanes
- Throw Event
- Text Annotation
- Message FLow

What we do not yet support in Open-BPMN are the elments

- Groups
- Sub Process
- Loop Process
- Data Store
- Data Input/Output
- Conversation Elements
- Choreography Elements


[Open-BPMN](https://github.com/imixs/open-bpmn) is open Source and missing Elements can be added by contributions
